extensions/typescript-language-features/package.json

```diff
@@ -301,6 +301,12 @@
           "markdownDescription": "%configuration.inlayHints.variableTypes.enabled%",
           "scope": "resource"
         },
+        "typescript.inlayHints.variableTypes.supressWhenTypeMatchesName": {
```

Comment:
```suggestion
        "typescript.inlayHints.variableTypes.suppressWhenTypeMatchesName": {
```

Reply:
Needs a `supress` -> `suppress` in several spots throughout this PR, not just here

Reply:
Good catch!